### ISSUE TITLE
Feature Request: Alphabetize destructured argument objects

### ISSUE THREAD

**Issue by reggi** [prettier/prettier#766]
Thoughts on coercing destructured argument objects to be in alphabetical order?  

Turning:

```
function example ({beta, alpha}) {

}
```

Into:

```
function example ({alpha, beta}) {

}
```

**Comment by bakkot** [comment]
That changes program semantics:

```js
const o = {get a(){console.log('a');}, get b(){console.log('b');}};
(({b, a}) => {})(o);
```
prints `b, a`, whereas

```js
(({a, b}) => {})(o);
```
prints `a, b`.

**Comment by lydell** [comment]
Yes, I think this has to be closed for the same reason as #323. ESLint might have a rule for it, though!
